examples/vkcs-lb-loadbalancer-create/main.tf

```diff
@@ -0,0 +1,10 @@
+resource "vkcs_lb_loadbalancer" "loadbalancer" {
+  name = "loadbalancer"
+  vip_subnet_id = "${vkcs_networking_subnet.lb.id}"
+  tags = ["tag1"]
+  timeouts {
+    create = "15m"
+    update = "15m"
+    delete = "15m"
+  }
+}
```

Comment:
we need much more elements of loadbalancer

Reply:
Added listener, pool, member